pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -287,8 +292,81 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(request reconcile.Request) (rec
 			return reconcile.Result{RequeueAfter: timeout}, nil
 		}
 
+		cnsFileVolumeClientInstance, err := cnsfilevolumeclient.GetFileVolumeClientInstance(ctx)
+		if err != nil {
+			msg := fmt.Sprintf("Failed to get CNSFileVolumeClient instance. Error: %+v", err)
+			log.Error(msg)
+			setInstanceError(ctx, r, instance, msg)
+			return reconcile.Result{RequeueAfter: timeout}, nil
+		}
+		err = cnsFileVolumeClientInstance.AddClientVMToIPList(ctx, instance.Namespace+"/"+instance.Spec.PvcName, instance.Spec.VMName, vm.Status.VmIp)
```

Comment:
Before this step, we need to do the following:
- Identify the network provider (NSX-T or VDS).
- Find external facing IP of the VM depending on NSXT or VDS
- Configure ACL with IP if needed. 

Since `CnsFileVolumeClients` is just used for bookkeeping, this step occurs after ACL configuration succeeds..

Reply:
Also, `AddClientVMToIPList` should take the IP determined from the steps above, not `vm.Status.vmIP`

Reply:
My bad, fixed this